src/main/java/gov/cabinetofice/gapuserservice/web/LoginControllerV2.java

```diff
@@ -75,6 +81,12 @@ public class LoginControllerV2 {
     @Value("${feature.onelogin.migration.enabled}")
     public String migrationEnabled;
 
+    @GetMapping("/validateAdminSession")
+    public ResponseEntity<Boolean> validateAdminRoles(@RequestHeader("emailaddress") String emailAddress, @RequestHeader("roles") String roles){
```

Comment:
Any reason these are requestHeaders and not request parameters?

Reply:
It came through as `emailaddress` when it was passed via headers in camelcase, have switched to query params & camel case does suit now